_includes/payment-resource.md

```diff
@@ -34,17 +34,19 @@ Content-Type: application/json
         "updated": "2016-09-14T13:21:57.6627579Z",
         "state": "Ready",
         "operation": "Purchase",
-        "intent": "Authorization",
-        "currency": "NOK",
+        "intent": "Authorization", {% if documentation_section == "vipps" %}
```

Comment:
Better to `assign` a `currency` and `language` variable at the top of the file and use them in the code example. `if` inside code examples are only necessary when entire parts of the code example should be conditionally shown or hidden.